winrm/tests/test_protocol.py

```diff
@@ -40,6 +58,32 @@ def test_get_command_output(protocol_fake):
     protocol_fake.close_shell(shell_id)
 
 
+def test_get_command_output_raw(protocol_fake):
+    shell_id = protocol_fake.open_shell()
+    command_id = protocol_fake.run_command(shell_id, "ipconfig", ["/all"])
+    std_out, std_err, status_code, done = protocol_fake.get_command_output_raw(shell_id, command_id)
+    assert status_code == 0
+    assert b"Windows IP Configuration" in std_out
+    assert len(std_err) == 0
+    assert done is True
+
+    protocol_fake.cleanup_command(shell_id, command_id)
+    protocol_fake.close_shell(shell_id)
+
+
+def test_get_command_output_raw_compat_api(protocol_fake):
+    shell_id = protocol_fake.open_shell()
+    command_id = protocol_fake.run_command(shell_id, "ipconfig", ["/all"])
+    std_out, std_err, status_code, done = protocol_fake._raw_get_command_output(shell_id, command_id)
```

Comment:
Maybe just parametrize the test and use `getattr` for the two values so you don't have to duplicate the rest of the test?

Reply:
Done